isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1835,6 +1839,36 @@
             <item>no</item>
           </default>
         </parameter>
+
+        <parameter name="ADJUSTMENTOUT_H5">
```

Comment:
Just to be more consistent with how ISIS names parameters, for better or worse, maybe name this `OutAdjustmentH5`